src/components/DataViewFilterMultiselect.jsx

```diff
@@ -116,16 +116,23 @@ function DataViewFilterMultiselect(props) {
 	const comparison = props.data.comparison || "in";
 	
 	useEffect(() => {
+		let isMouting = true;
 		async function fetchData() {
 			const selected = await props.args.getSelected(value);
-			
+
 			setState({
 				...state,
 				selected
 			});
 		}
 		
-		fetchData();
+		if(isMouting) {

```

Comment:
Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9783882)
```suggestion
		if (isMouting) {
```